mkfs.xfs: add page (#13704)
tldr-pages/tldr: pages/linux/mkfs.xfs.md

# mkfs.xfs

> Create an xfs filesystem inside a partition.
> More information: <https://manned.org/mkfs.xfs>.

> - Create an xfs filesystem inside partition 1 on device b (`sdb1`):

Reviewer: ```suggestion
- Create an xfs filesystem inside partition 1 on a device (`X`):
```

> `sudo mkfs.xfs {{/dev/sdb1}}`

Reviewer: ```suggestion
`sudo mkfs.xfs {{/dev/sdX1}}`
```

> > Create an xfs filesystem inside a partition.
> > More information: <https://manned.org/mkfs.xfs>.
> 
> - Create an xfs filesystem inside partition 1 on a device (`X`):
> 
> `sudo mkfs.xfs {{/dev/sdX1}}`
> 
> - Create an xfs filesystem with a volume-label:
> 
> `sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`

Reviewer 2: XFS is written in capital letters in the man page.

```suggestion
> Create an XFS filesystem inside a partition.
> More information: <https://manned.org/mkfs.xfs>.

- Create an XFS filesystem inside partition 1 on a device (`X`):

`sudo mkfs.xfs {{/dev/sdX1}}`

- Create an XFS filesystem with a volume label:

`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`
```

> `sudo mkfs.xfs -L {{volume_label}} {{/dev/sdb1}}`

Reviewer: ```suggestion
`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases
